contact.html

```diff
@@ -0,0 +1,114 @@
+        <center>
+            <h3><b>Let's Connect ☎️</b></h3>
+        </center>
+        <div id="form">
+
+            <div class="fish" id="fish"></div>
+            <div class="fish" id="fish2"></div>
+
+            <form id="waterform" method="post">
+
+                <div class="formgroup" id="name-form">
+                    <label for="name"><b>Your name</b><span style="color: red">*</span></label>
+                    <input type="text" id="name" name="name" maxlength="50" />
+                </div>
+
+                <div class="formgroup" id="email-form">
+                    <label for="email"><b>Your email</b><span style="color: red">*</span></label>
+                    <input type="email" id="email" name="email" maxlength="50" />
+                </div>
+
+                <div class="formgroup" id="message-form">
+                    <label for="message"><b>Your message</b></label>
+                    <textarea id="message" name="message" maxlength="5000"></textarea>
+                    <p style="text-align: left;">Note: <span style="color: red"> *</span>are
+                        Required fields </p>
+                </div>
+
+                <input id="submitButton" type="submit" value="Send your message!" />
+
+            </form>
+
+
+        </div>
+    </div>
+    <br><br>
+    <footer class="page-footer orange">
+        <div class="container">
+            <div class="row">
+                <div class="col l6 s12">
+                    <!-- donation button goes here -->
+                    <a href="https://act.autismspeaks.org/site/Donation2?idb=135287805&4141.donation=form1&DONATION_LEVEL_ID_SELECTED=1&df_id=4141&mfc_pref=T&idb=[[S76:idb]]"
+                        button class="btn waves-effect waves-light" type="submit" name="action">Donate to Autism
+                        Speaks
+                        <i class="material-icons right">monetization_on</i>
+                        </button></a>
+                    <a id="downloadLink" download="mediarecorder.webm" name="mediarecorder.webm" href></a>
+                </div>
+            </div>
+
+    </footer>
+
+    <!--  Scripts-->
+    <!-- <script src="https://code.jquery.com/jquery-2.1.1.min.js"></script> -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+    <!-- <video controls id="anotherVideo"></video> -->
+
+    <!-- The core Firebase JS SDK is always required and must be listed first -->
+    <!-- <script src="https://www.gstatic.com/firebasejs/5.11.1/firebase-app.js"></script>
+    <script src="https://www.gstatic.com/firebasejs/5.11.1/firebase-storage.js"></script> -->
+    <script src="https://www.gstatic.com/firebasejs/5.11.1/firebase.js"></script>
+    <!-- TODO: Add SDKs for Firebase products that you want to use
+     https://firebase.google.com/docs/web/setup#config-web-app -->
+    <script src="./js/contact.js"></script>
+    <script src="./js/contact-firebase.js"></script>
```

Comment:
take a look at master's firebaseInit.js
also link firebase init before any other js files
you'll need userInfo.js included in this, see my other comment below

Reply:
Resolved.